README.md

```diff
@@ -133,7 +136,6 @@ It currently
 
 To trigger a build, simply use `make static_files_process`
 
----
 ## Deploying prototypes
```

Comment:
I presume that the rewrite of this section will happen later.

Reply:
Yeah it still works for now, but when we move to AWS, it will be changed.